verification.py

```diff
@@ -1269,30 +1263,43 @@ def _msms_params_to_run_msp(self, params):
                     position=params["sel_pos"],
```

Comment:
Just realized this is another source of differences: for selection position, msms/discaol use a value between 0-1, msprime use position in bp.
After correcting this, it seems much better. This is `test_selective_discoal_vs_msp_use_discoal_paper_param` result:

before:
![image](https://user-images.githubusercontent.com/13037898/112497781-49d2ee00-8d5c-11eb-863c-2d6e7d8192e2.png)

and  after scaling the position

![image](https://user-images.githubusercontent.com/13037898/112497657-345dc400-8d5c-11eb-92db-7f8751d660e4.png)